modules/react-components/src/components/grid/resource-grid/resource-grid.tsx

```diff
@@ -97,6 +97,27 @@ export const ResourceGrid: FunctionComponent<
             wrapperClassName
         );
 
+        if (isLoading) {
+            return (
+                <Card.Group style={ { marginBottom: "3rem" } }>
+                    {
+                        [ ...Array(4) ].map(() => {
```

Comment:
Let's move the `4` to a constant in the component file itself.
```suggestion
                        [ ...Array(4) ].map(() => {
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3653/commits/4d9d7327a9f3509da0b1bc4492c8777dfdbec431 and https://github.com/wso2/identity-apps/pull/3653/commits/9b30a0d3fe7dd18a817cbc326811440d72e39f7b